modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -111,8 +111,8 @@ export default class Ecdsa {
         l: pShare.l,
         m: pShare.m,
         n: pShare.n,
-        y: bigIntToHex(y),
-        x: bigIntToHex(x),
+        y: y.toString(16),
```

Comment:
Is it okay that the expected output string is now a different syntax? Technically would we consider this a breaking change?

Reply:
This module is under active development and is not yet being used. It is okay to have it made.

Reply:
Awesome, thanks :)